server/app/app.py

```diff
@@ -67,6 +67,11 @@ def cache_control_always(**cache_kwargs):
     return _cache_control(True, **cache_kwargs)
 
 
+@webbp.errorhandler(RequestException)
+def handle_request_exception(error):
```

Comment:
Removing
```
@webbp.errorhandler(TombstoneException)
def handle_tombstone_exception(error):
    return redirect(f"{current_app.app_config.server_config.get_web_base_url()}/collections/{error.collection_id}?tombstoned_dataset_id={error.dataset_id}") # noqa E501
```

Still works. Doesn't seem necessary. `CellxgeneException` also is not "registered", but the errors defined with this exception class are still functional.